inlineplz/interfaces/base.py

```diff
@@ -12,3 +12,10 @@ def post_messages(self, messages, max_comments):
         :return:
         """
         raise NotImplementedError()
+
+    def clear_outdated_messages(self):
```

Comment:
`prospector: pep257: One-line docstring should fit on one line with quotes (found 4) (D200)`